.yarnrc.yml

```diff
@@ -1 +1,3 @@
-nodeLinker: node-modules
\ No newline at end of file
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.2.0.cjs
```

Comment:
This ensures yarn.lock produced in this repo matches the one produced when running install in monorepo